x/leveragelp/keeper/msg_server_claim_rewards.go

```diff
@@ -16,6 +16,14 @@ func (k msgServer) ClaimRewards(goCtx context.Context, msg *types.MsgClaimReward
 		if err != nil {
 			return nil, err
 		}
+
+		// Add trigger function
+		pool, _ := k.GetPool(ctx, position.AmmPoolId)
+		_, _, _, err = k.CheckAndLiquidateUnhealthyPosition(ctx, &position, pool)
```

Comment:
if the position is closed full then masterchefKeeper.ClaimRewards is being called twice

Reply:
add a check